janus/src/config.c

```diff
@@ -53,8 +53,21 @@ us_config_s *us_config_init(const char *config_dir_path) {
 	}
 	if ((config->audio_dev_name = _get_value(jcfg, "audio", "device")) != NULL) {
 		if ((config->tc358743_dev_path = _get_value(jcfg, "audio", "tc358743")) == NULL) {
-			US_JLOG_INFO("config", "Missing config value: audio.tc358743");
-			goto error;
+			if((config->pcm_path = _get_value(jcfg, "audio", "pcm_path")) == NULL) {
```

Comment:
Please use just `"pcm"` for the config option name: `config->pcm_path = _get_value(jcfg, "audio", "pcm")`

Also the space required after `if`.